docs/utils/util_data_structures.rst

```diff
@@ -65,6 +65,20 @@ Exact Library
 .. doxygenclass:: mockturtle::exact_library
    :members:
 
+Super Utils
```

Comment:
From this name, it's not very easy to know what the data structure is for. Maybe "Super Gates" would fit better at least for the title?